packages/webapp/src/utils/credits-purchase-constants.js

```diff
@@ -1,12 +1,20 @@
 const routes = {
-  ESTIMATOR_CREDITS_INDIVIDUAL_ORG: '/credits-purchase/applying-individual-organisation'
+  APPLYING_INDIVIDIAL_ORGANISATION: '/credits-purchase/applying-individual-organisation',
+  CREDITS_PURCHASE_CHECK_DEFRA_ACCOUNT_DETAILS: '/credits-purchase/check-defra-account-details'
+  // REGISTER_CREDIT_PURCHASE_TASK_LIST: '/credit-purchase/register-credit-purchase-task-list'
 }
 
 const views = Object.fromEntries(
   Object.entries(routes).map(([k, v]) => [k, v.substring(1)])
 )
 
+const redisKeys = {
+  APPLYING_INDIVIDIAL_ORGANISATION: 'applying-individual-organisation',
+  DEFRA_ACCOUNT_DETAILS_CONFIRMED: 'defra-account-details-confirmed'
```

Comment:
The landowner and developer journeys have Redis key constants with journey specific values.

As landowner was the original journey, its non-route constant names and values have no journey specific prefix generally. Developer journey non-route constant names are prefixed with DEVELOPER_ and the constant values are prefixed with developer- generally. As for other credits purchase journey constants I think non-route constant names for the credits purchase journey should be prefixed with CREDITS_PURCHASE_  and the values should be prefixed with credits-purchase-.

Reply:
done